### DISCUSSION TITLE
Why there are "name" and "username" in the "users" collection

### DISCUSSION THREAD

**Discussion by Tilican** [pocketbase/pocketbase#4504] [Q&A] [answered]
Hey,

Thanks for this project, I really enjoy using it!

As part of my sideproject I would like to make a user search system to add him to a group.

I have modified the users rules so that any user can retrieve a list of users following a name ~ "user3" for example

But what's the use of "username" (except for login) compared to "name" since the 2 go back up? 

In any case, it's only the email that doesn't go up, so do you need a "usernameVisibility"? Because at the moment, if you want to activate the "username" as a login, anyone can know each usernames and it's easier to bruteforce account.

Maybe it's better to make a "profile" collection linked to a user with "name" and "avatar" so as not to "open" the "users" collection? 

Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
The "name" is just a regular text field and you can delete it if you are not using it. It is intended to hold values like "John Doe" (aka. the full name of the user).

The "username" is a special system field used mainly for auth and at the moment it cannot be removed or customized (this will change with the refactoring). It is intended to hold values like "john.doe" (aka. the username/nickname of the user).

> Maybe it's better to make a "profile" collection linked to a user with "name" and "avatar" so as not to "open" the "users" collection?

This was discussed before in https://github.com/pocketbase/pocketbase/issues/376.

**Reply by Tilican** [reply]
OK, nice !
Thx again @ganigeorgiev
